hera_sim/simulate.py

```diff
@@ -434,4 +435,64 @@ def add_xtalk(self, model='gen_whitenoise_xtalk', bls=None, **kwargs):
                 vis=self.data.data_array[blt_ind, 0, :, pol_ind],
                 xtalk=xtalk
             )
+    
+    def make_data(self, sim_file=None, sim_params=None):
```

Comment:
if you want you can make this just `make_data(self, sim_params)`
and do
```
if isinstance(sim_params (str, np.str)):
    with open(sim_params, 'r') as doc:
        sim_params = yaml.load(doc.read(), Loader=yaml.FullLoader)
```

Reply:
I think the way that @r-pascua has done it makes sense (except backwards, see my other comment) -- it may be sometimes desired to overwrite the file parameters with dynamic ones.